### PR TITLE
Bugfixes for TV Show banner layout

### PR DESCRIPTION
**Pull request by wutschel**
## Description

Use same background for `contenView` and `cell` for TV Show banners.
Set `heightForHeaderInSection` to 0 when there is only 1 section.

Screenshots:
<a href="https://abload.de/image.php?img=bildschirmfoto2023-05xlizs.png"></a>

## Summary for release notes

Bugfix: Remove white line on top of TV Show banners
Bugfix: Remove white frame around selected TV Show banner

### CODE REVIEW

File: XBMC Remote/DetailViewController.m [hunk 1]
```diff
@@ -3130,7 +3133,7 @@ - (CGFloat)tableView:(UITableView*)tableView heightForHeaderInSection:(NSInteger
         return sectionHeight;
     }
     if ([self.sections allKeys].count == 1) {
-        return 1;
+        return 0;
```
Comment (kambala-decapitator) [inline XBMC Remote/DetailViewController.m:3136]: looks like this condition can be simply removed

File: XBMC Remote/DetailViewController.m [hunk 2]
```diff
@@ -921,6 +921,9 @@ - (void)layoutTVShowCell:(UIView*)cell useDefaultThumb:(BOOL)useFallback imgView
                 cell.backgroundColor = [Utilities getSystemGray6];
             }
         }
+        if ([cell isKindOfClass:[UITableViewCell class]]) {
+            ((UITableViewCell*)cell).contentView.backgroundColor = cell.backgroundColor;
```
Comment (kambala-decapitator) [inline XBMC Remote/DetailViewController.m:925]: FYI to reduce number of () to make it slightly more readable:
```suggestion
            [(UITableViewCell*)cell contentView].backgroundColor = cell.backgroundColor;
```

**Review by kambala-decapitator — APPROVED** [review]

File: XBMC Remote/DetailViewController.m (hunk 1 above)
Comment (wutschel) [reply XBMC Remote/DetailViewController.m]: done

File: XBMC Remote/DetailViewController.m (hunk 2 above)
Comment (wutschel) [reply XBMC Remote/DetailViewController.m]: done

**Comment by wutschel** [comment]
Updated the commits with the suggested changes.

**Pull request merged**